core/sdk/src/extn/client/extn_client.rs

```diff
@@ -655,18 +656,29 @@ impl ExtnClient {
         if !self.sender.get_cap().is_main() {
             return Err(RippleError::InvalidAccess);
         }
+        debug!("Main internal request with payload: {:?}", payload);
```

Comment:
Do we need these DEBUG level logs ? Will this contribute to log bloating ?

Reply:
Updated to trace, although these would only be for non prod builds